docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -90,17 +106,15 @@ Here is a fully working example:
 
 ## Toggling color mode
 
-You can use the React context to toggle the mode with a button inside your page.
+If you want to provide your users a way to toggle between modes, you can use React's context adding it to a button's `onClick` event:
 
 {{"demo": "ToggleColorMode.js", "defaultCodeOpen": false}}
 
 ## System preference
 
-Users might have specified a preference for a light or dark theme.
-The method by which the user expresses their preference can vary. It might be a system-wide setting exposed by the Operating System, or a setting controlled by the User Agent.
+The method by which users express their mode preference can vary. It might be a system-wide setting exposed by the Operating System, or a setting controlled by the User Agent (user specifying a preference for a light or dark theme).
 
 You can leverage this preference dynamically with the [useMediaQuery](/components/use-media-query/) hook and the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query.
-
 For instance, you can enable the dark mode automatically:
```

Comment:
This part here got me confused. What I understood from the first topic was that having the `createTheme` with `mode: 'dark'` would set the dark mode as the default one for the app (hence the title I added). But here it seems that we're talking about the same thing?! What is the actual difference?

Reply:
If I'm understanding correctly:

- in the earlier section, it's talking about using dark mode as the default for the app, regardless of the user's preference
- in this part, it's showing how to check if the user prefers dark mode (whether through OS or browser settings), and then enabling/disabling based on that preference

I think you're right that we should do some more clarification.